src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -257,7 +257,7 @@ const ANNOTATION_ELEMENT_ID = 'mutation-annotation';
 export default class MutationTable<
     P extends IMutationTableProps
 > extends React.Component<P, {}> {
-    @observable protected _columns: Record<
+    @observable.ref protected _columns: Record<
```

Comment:
@onursumer this could screw things up if we are actually mutating columns somewhere.   we are toggling the visible boolean but it seems that toggling visibility still works

Reply:
Looks like column filters are not working anymore though 😞

Reply:
@onursumer good find. thanks.  will try to fix